src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSelector.java

```diff
@@ -0,0 +1,268 @@
+            }
+            ((RemoteBitrateEstimator)packetTransformer)
+                    .setMinBitrate(minBitrateBps);
+        }
+    }
+
+
+    @Override
+    public Collection<Integer> getSsrcs()
+    {
+        return ((RemoteBitrateEstimator)packetTransformer).getSsrcs();
+    }
+
+    /**
+     * Returns the estimated payload bitrate in bits per second if a valid
+     * estimate exists; otherwise, <tt>-1</tt>.
+     *
+     * @return the estimated payload bitrate in bits per seconds if a valid
+     * estimate exists; otherwise, <tt>-1</tt>
+     */
+    @Override
+    public long getLatestEstimate()
+    {
+
+        return ((RemoteBitrateEstimator)packetTransformer).getLatestEstimate();
+    }
+
+    /**
+     * Removes all data for <tt>ssrc</tt>.
+     *
+     * @param ssrc
+     */
+    @Override
+    public void removeStream(int ssrc) {
+        ((RemoteBitrateEstimator)packetTransformer).removeStream(ssrc);
+    }
+
+    @Override
+    public void setMinBitrate(int minBitrateBps) {
+        synchronized (critSect)
+        {
+            ((RemoteBitrateEstimator)packetTransformer)
+                    .setMinBitrate(minBitrateBps);
+            this.minBitrateBps = minBitrateBps;
+        }
+    }
+
+    @Override
+    public RawPacket reverseTransform(
+            RawPacket packet)
+    {
+        /**
+         * @Todo Find out why having synchronized here makes sense since it
+         * exists in PickEstimatorFromHeader.
+         */
+        if (this.absSendTimeEngine == null){
+            this.absSendTimeEngine = stream.getAbsSendTimeEngine();
+        }
+
+        synchronized (critSect)
+        {
+            if(enableUseOfAbsSendTime)
+            {
+                PickEstimatorFromHeader(packet);
+            }
+            packetTransformer.reverseTransform(packet);
+        }
+        return packet;
+    }
+
+    /**
+     * Gets the <tt>PacketTransformer</tt> for RTP packets.
+     *
+     * @return the <tt>PacketTransformer</tt> for RTP packets
+     */
+    @Override
+    public PacketTransformer getRTPTransformer() {
+        return this;
+    }
+
+    /**
+     * Gets the <tt>PacketTransformer</tt> for RTCP packets.
+     *
+     * @return the <tt>PacketTransformer</tt> for RTCP packets
+     */
+    @Override
+    public PacketTransformer getRTCPTransformer() {
+        return null;
+    }
+
+
+    /**
+     * Returns the number of milliseconds until this instance wants a worker
+     * thread to call {@link #run()}. The method is called on the same
+     * worker thread as Process will be called on.
+     *
+     * @return the number of milliseconds until this instance wants a worker
+     * thread to call {@link #run()}
+     */
+    @Override
+    public long getTimeUntilNextRun() {
+        long timeUntilNextRun  =
+                (packetTransformer instanceof RecurringRunnable) ?
+             ((RecurringRunnable) packetTransformer)
+                     .getTimeUntilNextRun() : -1;
```

Comment:
Updated it to return 500Ms. I think it is a better choice. If RBEAbsSendTime should for some reason implement the getTimeUntilNextRun method in the future. Nothing will need to be changed in the Selector.